backend/src/main/java/com/crm/config/SecurityConfiguration.java

```diff
@@ -34,6 +34,7 @@
             "/webjars/**",
             // -- other
```

Comment:
W sumie możemy wszystkie endpointy związane z autentykacją wrzucić do oddzielnej zmiennej. Będzie to bardziej eleganckie i obejdzie się bez komentarzy.